WebRoot/js/zimbraAdmin/accounts/view/ZaAccountXFormView.js

```diff
@@ -3646,3 +3667,22 @@ ZaAccountXFormView.prototype.getTabChoices =
 function() {
     return this.tabChoices;
 }
+
+ZaAccountXFormView.getEmailFromUid = managerField => {
+	if (managerField) {
+		let email = '';
+		let managerLdapArray = managerField?.split(',');
```

Comment:
same as above for optional chaining operator

Reply:
changed